benchopt/tests/test_benchmarks/dummy_benchmark/solvers/solver_test.py

```diff
@@ -52,6 +52,8 @@ def st(self, w, mu):
     def get_result(self):
         return self.w
 
-    @staticmethod
-    def get_next(stop_val):
+    def get_next(self, stop_val):
+        # check that we access instance parameters
+        self.raise_error
```

Comment:
I would not bother to do this here, this will already fails if it is only for static as `self` would not be defined in the signature..
```suggestion
```